src/api/spec/services/notification_service/notifier_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe NotificationService::Notifier do
+  let(:user_bob) { create(:confirmed_user, login: 'bob') }
+  let(:user_kim) { create(:confirmed_user, login: 'kim') }
+  let(:commenter) { create(:confirmed_user, login: 'ann') }
+
+  let(:create_bob_subscription) { create(:event_subscription_comment_for_project, receiver_role: 'maintainer', user: user_bob, channel: :rss) }
```

Comment:
Rubocop complained about the previous `let!` statements that weren't referenced anywhere else in the spec. I've made it explicit in those cases were they were necessary, but I don't fully like how it looks like. The thing is the alternatives I have (use the factories inline where necessary) are very much zero DRY. Any ideas?